scripts/publish-npm.ts

```diff
@@ -150,7 +150,7 @@ async function main() {
     shell.cd(pkg);
     const otp =
         await question(`Enter one-time password from your authenticator: `);
-    $(`npm publish --otp=${otp}`);
+    $(`YARN_REGISTRY="https://registry.npmjs.org/" npm publish --otp=${otp}`);
```

Comment:
This doesn't work. I have tried this before. The env variable in the console effectively controls the env of the immediately following yarn command. However, shelljs doesn't seem to be able to override that env. So it has to be in the console. Somehow, weirdly, adding it in shelljs works on my macbook. But it doesn't work in my workstation. Having the env set in console works cross platforms.

Reply:
We can add this though, because it works in some platform. But we should advise people to also set the env in the console for now. If shellsjs can fix this problem in the future, then we can drop the outside part.

Reply:
good to know, interestingly it did work (i think) on my mac. I'll test futher and see if it works without this line.

Reply:
It works on my mac too. It doesn't work on gLinux.